harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -122,23 +122,67 @@ def _init_device(self) -> None:
             self.device = torch.device("cpu")
         check.is_not_none(self.device)
 
-    def _init_model_and_optimizer(self) -> None:
+    def _init_model_and_optimizers(self) -> None:
```

Comment:
blocking: function name should reflect that `lr_schedulers` are being configured here as well